doc/plugin_server_nodeattestor_aws_iid.md

```diff
@@ -45,6 +48,27 @@ In the following configuration,
 
 assuming AWS IID document sent from the spire agent contains `accountId : 12345678`, the spire server will assume "arn:aws:iam::12345678:role/spire-server-delegate" role before making any AWS call for the node attestation. If `assume_role` is configured, the spire server will always assume the role even if the both the spire-server and the spire agent is deployed in the same account.
 
+##### Enabling AWS Node Attestation Organization Validation
+
+For configuring AWS Node attestation method with organization validation following configuration can be used : 
+
+```hcl
+  NodeAttestor "aws_iid" {
+    plugin_data {
+      partition          = "aws"
+      assume_role        = "spire_node_attestor"
+      account_ids_belong_to_org_validation {
+        org_account_id     = "7891011"
```

Comment:
Must this always be the management account? Should we name it `management_account_id` or similar?